Kudu.Services/Diagnostics/LinuxProcessController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace Kudu.Services.Performance
+{
+    // This is a placeholder for future process API functionality on Linux,
+    // the implementation of which will differ from Windows enought that it warrants
+    // a separate controller class. For now this returns 400s for all routes.
+
+    public class LinuxProcessController : ApiController
+    {
+        private const HttpStatusCode RETURNCODE = HttpStatusCode.BadRequest;
```

Comment:
nit: This constant has limited value IMO. Code is more readable if you directly use `HttpStatusCode.BadRequest` everywhere.